docs/experimental-features.md

```diff
@@ -343,3 +343,23 @@ See [Plugin docs](./plugins.md)
 
 - [ ] Needs more testing
 - [ ] Make sure there are no unknown major problems
+
+## Sharding / HAMT
+
+### In Version
+0.4.8
+
+### State
+Experimental
+
+Currently, when too many items are added into a unixfs directory, the object gets too large and you may experience issues. To pervent this problem, and generally make working really large directories more efficient, we have implemented a HAMT structure for unixfs. To enable this feature, run:
```

Comment:
I'd keep the description short, something like: `Allows to create directories with unlimited number of entries - currently size of unixfs directories is limited by the maximum block size.`

Also, try to keep the lines in docs under 80 characters.